### PR TITLE
feat: Introudce default RPC provier and fix another issue

### PR DESCRIPTION
**Pull request by SamuraiT**
# objective
- to fix enable true issue
- modify index page

### CODE REVIEW

**Review by geeknees — APPROVED** [review]
It works!

Is it correct to ignore the following for the local environment and the production environment?

local(localhost:3000)
スクリーンショット 2022-05-12 12 36 56

production(https://henkaku-membership-git-add-de-14a7ca-henkaku-community-dev-team.vercel.app/mintKamon)
スクリーンショット 2022-05-12 12 33 12

We have to disable the `enable option` to enable the `watch option`, and I understand the warning and error on wallet connect once it is omit for now.

File: pages/index.tsx [hunk 1]
```diff
@@ -1,62 +1,72 @@
...
+          )}
+        </Text>
         <SimpleGrid columns={{ sm: 1, md: 1, lg: 2 }} spacing="10px">
           <div>
-            {tokenURIImage ? (
-              <Image src={tokenURIImage} alt="" />
-            ) : (
-              <Image src={'/kamonNFT_427@2x.png'} alt="" />
-            )}
+            <Image src={'/kamonNFT_427@2x.png'} alt="" />
```
Comment (hourglasshoro) [inline pages/index.tsx:47]: Is it like making it a fixed image once?

**Review by hourglasshoro — APPROVED** [review]
thank you very much for your support!

File: pages/index.tsx (hunk 1 above)
Comment (SamuraiT) [reply pages/index.tsx:47]: for now but I will modify here in diffrent PR

**Pull request merged**